meinberlin/apps/livequestions/assets/QuestionUser.jsx

```diff
@@ -69,18 +69,23 @@ export default class QuestionUser extends React.Component {
         <div className="live-question__action-bar">
           {this.props.hasLikingPermission
             ? (
-              <button type="button" className={(this.state.session_like ? 'u-success ' : '') + 'btn'} onClick={this.handleLike.bind(this)}>
-                <span>{this.state.likes} </span>
+              <button
+                type="button"
+                className={'rating-button rating-up ' + (this.state.session_like ? 'u-success ' : 'u-muted')}
+                onClick={this.handleLike.bind(this)}
+                aria-label={this.state.session_like ? addLikeTag : undoLikeTag}
```

Comment:
we should maybe consider using a component for our like button as we now have it twice

Reply:
Is this like the support button that we also use for the 3 pahse budgeting?